docsite/rst/YAMLSyntax.rst

```diff
@@ -59,6 +77,17 @@ Ansible doesn't really use these too much, but you can also specify a boolean va
     likes_emacs: TRUE
     uses_cvs: false
 
+Values can span multiple lines using *|* or *>* to include newlines or ignore them::
```

Comment:
This should include a brief description on the difference between the two syntaxes.